src/Updater.php

```diff
@@ -498,7 +500,7 @@ protected function validatePhar($phar)
 
     protected function cleanupAfterError()
     {
-        //@unlink($this->getBackupPharFile());
+        @unlink($this->getBackupPharFile());
```

Comment:
all the files don't necessarily exists so we need to unlink them only if they exists no?